src/CommonITILTask.php

```diff
@@ -1944,20 +1944,21 @@ public static function showVeryShort($ID, $itemtype)
          </td>";
 
             echo "<td>";
-            echo $item_link->fields['name'];
+            echo htmlspecialchars($item_link->fields['name']);
             echo "</td>";
 
             echo "<td>";
-            $link = "<a id='" . strtolower($item_link->getType()) . "ticket" . $item_link->fields["id"] . $rand . "' href='" .
+            $link = "<a id='" . htmlspecialchars(strtolower($item_link->getType())) . "ticket" . htmlspecialchars($item_link->fields["id"] . $rand) . "' href='" .
                    $item_link->getFormURLWithID($item_link->fields["id"]);
-            $link .= "&amp;forcetab=" . $tab_name . "$1";
+            $link .= "&amp;forcetab=" . htmlspecialchars($tab_name) . "$1";
             $link   .= "'>";
             $link    = sprintf(__('%1$s'), $link);
-            printf(
+            $link = sprintf(
                 __('%1$s %2$s'),
                 $link,
                 Html::resume_text(RichText::getTextFromHtml($job->fields['content'], false, true), 50)
```

Comment:
Html::resume_text() outputs HTML code, so it should not be escaped again.

Reply:
Fixed